src/styles/index.ts

```diff
@@ -5634,8 +5634,8 @@ const styles = (theme: ThemeColors) =>
         },
         reportPreviewArrowButton: {
             borderRadius: 50,
-            width: 28,
-            height: 28,
+            width: variables.h28,
+            height: variables.w28,
```

Comment:
Right?
```suggestion
            width: variables.w28,
            height: variables.h28,
```

Reply:
No way, I honestly didn't do this on purpose 😅 

Fixed.

Reply:
Unfortunately, I'm craving a McD burger, so no thanks.